src/PerformanceTests/Common/Scenarios/BaseRunner.cs

```diff
@@ -88,7 +88,12 @@ void CreateSeedData()
 
             var count = 0L;
 
-            Parallel.ForEach(IterateUntilFalse(() => !cts.Token.IsCancellationRequested), b =>
+            var po = new ParallelOptions
+            {
+                MaxDegreeOfParallelism = ConcurrencyLevelConverter.Convert(Permutation.ConcurrencyLevel)
```

Comment:
I am pretty sure this setting is actually by default the number of cores, so if `Permutation.ConcurrencyLevel` is higher then the number of cores, you will actually see a degradation in performance, see https://msdn.microsoft.com/en-us/library/system.threading.tasks.paralleloptions.maxdegreeofparallelism(v=vs.110).aspx#Anchor_2